docs/fsharp/get-started/get-started-vscode.md

```diff
@@ -129,63 +115,49 @@ Check if the first character of the input word is a vowel. If it is, attach "yay
-
-1. How to set up Visual Studio Code with Ionide.
-2. How to create your first F# project with Ionide.
-3. How to use F# Scripting to write your first F# function in Ionide and then execute it in FSI.
-4. How to migrate your script code to F# source and then call that code from FSI.
-
-You're now equipped to write much more F# code using Visual Studio Code and Ionide.
-
-## Troubleshooting
+## Troubleshooting Ionide
 
 Here are a few ways you can troubleshoot certain problems that you might run into:
 
 1. To get the code editing features of Ionide, your F# files need to be saved to disk and inside of a folder that is open in the Visual Studio Code workspace.
-2. If you've made changes to your system or installed Ionide prerequisites with Visual Studio Code open, restart Visual Studio Code.
-3. Check that you can use the F# compiler and F# interactive from the command line without a fully-qualified path. You can do so by typing `fsc` in a command line for the F# compiler, and `fsi` or `fsharpi` for the Visual F# tools on Windows and Mono on Mac/Linux, respectively.
-4. If you have invalid characters in your project directories, Ionide might not work.  Rename your project directories if this is the case.
-5. If none of the Ionide commands are working, check your [Visual Studio Code keybindings](https://code.visualstudio.com/docs/customization/keybindings#_customizing-shortcuts) to see if you're overriding them by accident.
-6. If Ionide is broken on your machine and none of the above has fixed your problem, try removing the `ionide-fsharp` directory on your machine and reinstall the plugin suite.
-
-Ionide is an open source project built and maintained by members of the F# community. Please report issues and feel free to contribute at the [Ionide-VSCode: FSharp GitHub repository](https://github.com/ionide/ionide-vscode-fsharp).
+1. If you've made changes to your system or installed Ionide prerequisites with Visual Studio Code open, restart Visual Studio Code.
+1. If you have invalid characters in your project directories, Ionide might not work.  Rename your project directories if this is the case.
+1. If none of the Ionide commands are working, check your [Visual Studio Code keybindings](https://code.visualstudio.com/docs/customization/keybindings#_customizing-shortcuts) to see if you're overriding them by accident.
+1. If Ionide is broken on your machine and none of the above has fixed your problem, try removing the `ionide-fsharp` directory on your machine and reinstall the plugin suite.
 
-If you have an issue to report, please follow [the instructions for getting logs to use when reporting an issue](https://github.com/ionide/ionide-vscode-fsharp#how-to-get-logs-for-debugging--issue-reporting).
+Ionide is an open source project built and maintained by members of the F# community. Please report issues and feel free to contribute at the [ionide-vscode-fsharp GitHub repository](https://github.com/ionide/ionide-vscode-fsharp).
```

Comment:
Mentioning Open Collective would be too much, wouldn’t it?

Reply:
Probably, yes.